day_7/caesar.rb

```diff
@@ -0,0 +1,34 @@
+#
+# def to_code(string)
+# encode = 26
+# code = encode.chars.map(&:ord)
+# code = encode.chars.map {|x| x.ord}
+# moving = encode.map {|x| x + 3}
+# moving.map {|x| x.chr }.join
+# end
+#
+# puts to_code("ouch")
+
+def caesar_cipher(k)
+  alphabet = Array('a'..'z')
+  encrypter = Hash[alphabet.zip(alphabet.rotate(3))]
+  k.split('').map { |x| encrypter.fetch(x, " ") }
```

Comment:
Very nice work creating a hash with the enumerator .zip and using `alphabet.rotate` then using .map on the next line to reach into the hash and create an encrypted array. There's a lot happening on these two lines!